src/viur_cli/conf.py

```diff
@@ -147,10 +175,15 @@ def migrate(self):
             )
 
             if response == "yes":
-                del self["default"]["builds"]["vi"]
+                try:
+                    del self["default"]["builds"]["vi"]
+                except:
+                    echo_info("You are using the ViUR Admin")
```

Comment:
This doesn't make sense to me. Why is the echo_info only printed when an exception occurs?

My proposal:
```suggestion
                self["default"]["builds"].pop("vi", None)
                echo_info("You are using the ViUR Admin")
```